docs/build-docs

```diff
@@ -67,7 +67,7 @@ def build_module_docs() -> None:
             "-D",
             f"project=pytket-{MODULE}",
             "-D",
-            f"copyright={datetime.date.today().year} Cambridge Quantum Computing",
+            f"copyright={date.today().year} Cambridge Quantum Computing",
```

Comment:
Can we remove/update the CQC copyright?

Reply:
Not in this PR. Note that this is a merge to _main_ from a branch off _develop_ -- please just review the last commit.

Reply:
We should keep in mind to do this